### ISSUE TITLE
Unable to manually enter otpauth:// URL

### ISSUE THREAD

**Issue by rmonk** [freeotp/freeotp-android#161]
I have a phone with a broken camera, so QR codes are not easily available for it.  I noticed that if you manually enter an otpauth:// URI to get it to load it in FreeOTP it does not bounce over to the app as you would expect.  Can it register itself as a handler for those URIs so manual entry or copy/paste would work?

**Comment by afattahi54** [comment]
Did you solve this issue ?!

**Comment by rmonk** [comment]
No, this continues to be a problem.  I haven't seen any updates to this at all.

**Comment by npmccallum** [comment]
I'll look at this today.

**Comment by dlenski** [comment]
> Can it register itself as a handler for those URIs so manual entry or copy/paste would work?

I've been wondering about this as well. The app clearly has all the machinery to consume `otpauth://` URLs, since it handles them fine via QR codes… but it doesn't associate itself with them as an intent handler.

@npmccallum, do you have any pre-built APKs of your rewrite?

**Comment by rmonk** [comment]
Are there any updates for this issue?

**Comment by adambkaplan** [comment]
@npmccallum is this a use case we want to support? Given the recent changes it `master` it seems we only support reading QR codes via the camera.
